core/services/ocr2/plugins/functions/plugin.go

```diff
@@ -57,6 +57,8 @@ const (
 	FunctionsS4Namespace                  string = "functions"
 	MaxAdapterResponseBytes               int64  = 1_000_000
 	DefaultOffchainTransmitterChannelSize uint32 = 1000
+	DefaultMaxAdapterRetry                int    = 3
+	DefaultExponentialBackoffBase                = 1 * time.Second
```

Comment:
I'd start with a higher default, maybe 5 seconds? Also what happens if the request (computation) takes a long time? Is retry time measured from when previous response happened?